src/v/transform/rpc/client.cc

```diff
@@ -807,6 +808,38 @@ client::generate_remote_report(model::node_id node) {
     co_return std::move(resp).value().report;
 }
 
+ss::future<cluster::errc> client::create_transform_logs_topic() {
+    co_return co_await retry(
+      [this]() { return try_create_transform_logs_topic(); });
+}
+
+ss::future<cluster::errc> client::try_create_transform_logs_topic() {
+    cluster::topic_properties topic_props;
+    topic_props.batch_max_bytes
+      = config::shard_local_cfg().kafka_batch_max_bytes();
```

Comment:
Isn't this already the default if not specified? Should we just leave it unset to use the (dynamic) cluster default?

Reply:
done